### PR TITLE
create navbar component with style #40

### PR DESCRIPTION
**Pull request by mohammedsalah7**
Screenshot from 2021-11-01 16-57-16

### CODE REVIEW

File: client/src/components/Navbar/index.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+import {
+  ListItem, ListItemText, Button, Avatar, Container, List,
+} from '@mui/material';
+import './style.css';
+
+import Logo from '../../asstes/logo.png';
+import PresonImg from '../../asstes/avatar.png';
+
+function NavBar() {
+  const [state, setstate] = useState(true);
```
Comment (Mu7ammadAbed) [inline client/src/components/Navbar/index.js:14]:
```suggestion
  const [logged, setLogged] = useState(true);
```

File: client/src/components/Navbar/index.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
+/* eslint-disable no-unused-vars */
+import React, { useState } from 'react';
+import { Link as RouterLink } from 'react-router-dom';
+import Link from '@mui/material/Link';
```
Comment (Mu7ammadAbed) [inline client/src/components/Navbar/index.js:4]: There's no need to use the the `react-router-dom' Link inside of `material-ui` Link
```suggestion
import { Link } from 'react-router-dom';
```

File: client/src/components/Navbar/index.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+import Logo from '../../asstes/logo.png';
+import PresonImg from '../../asstes/avatar.png';
+
+function NavBar() {
+  const [logged, setLogged] = useState(true);
+  return (
+    <Container maxWidth="xl">
+      <nav className="container">
+        <Link component={RouterLink} to="/">
```
Comment (Mu7ammadAbed) [inline client/src/components/Navbar/index.js:18]:
```suggestion
        <Link to="/">
```

File: client/src/components/Navbar/index.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+        </Link>
+        {' '}
+        <List
+          className="list"
+          sx={{ justifyContent: 'space-between' }}
+          style={{
+            marginRight: '150px', alignSelf: 'center', color: '#797B7D',
+          }}
+        >
+          <Link component={RouterLink} to="/" style={{ color: '#797B7D', textDecoration: 'none' }}>
```
Comment (Mu7ammadAbed) [inline client/src/components/Navbar/index.js:29]:
```suggestion
          <Link to="/" style={{ color: '#797B7D', textDecoration: 'none' }}>
```

File: client/src/components/Navbar/index.js [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+            <ListItem button style={{ textDecoration: 'none' }}>
+              <ListItemText primary="Home" />
+            </ListItem>
+          </Link>
+
+          <Link component={RouterLink} to="/signup" style={{ color: '#797B7D', textDecoration: 'none' }}>
```
Comment (Mu7ammadAbed) [inline client/src/components/Navbar/index.js:35]:
```suggestion
          <Link to="/signup" style={{ color: '#797B7D', textDecoration: 'none' }}>
```

File: client/src/components/Navbar/index.js [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+            <ListItem button>
+              <ListItemText primary="About" />
+            </ListItem>
+          </Link>
+          <Link component={RouterLink} to="/" style={{ color: '#797B7D', textDecoration: 'none' }}>
```
Comment (Mu7ammadAbed) [inline client/src/components/Navbar/index.js:40]:
```suggestion
          <Link to="/" style={{ color: '#797B7D', textDecoration: 'none' }}>
```

File: client/src/components/Navbar/index.js [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+            <ListItem button>
+              <ListItemText primary="Buy" />
+            </ListItem>
+          </Link>
+          <Link component={RouterLink} to="/" style={{ color: '#797B7D', textDecoration: 'none' }}>
```
Comment (Mu7ammadAbed) [inline client/src/components/Navbar/index.js:45]:
```suggestion
          <Link to="/" style={{ color: '#797B7D', textDecoration: 'none' }}>
```

File: client/src/components/Navbar/index.js [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+            <ListItem button>
+              <ListItemText primary="Rent" />
+            </ListItem>
+          </Link>
+        </List>
+        {/* </Stack> */}
```
Comment (Mu7ammadAbed) [inline client/src/components/Navbar/index.js:51]: ??
```suggestion
```

File: client/src/components/Navbar/index.js [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+
+        <div className="agent">
+          <Button style={{
+            marginRight: '30px', alignSelf: 'center', backgroundColor: '#3781CB', color: '#FFFFFF', textTransform: 'none',
+          }}
+          >
+            {' '}
+            <Link component={RouterLink} to="/login" className="btn-host">
```
Comment (Mu7ammadAbed) [inline client/src/components/Navbar/index.js:59]:
```suggestion
            <Link to="/login" className="btn-host">
```

File: client/src/components/Navbar/index.js [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+              <ListItemText primary="Host my House" style={{ color: '#fff' }} />
+            </Link>
+          </Button>
+          {logged ? (
+            <Link component={RouterLink} to="/">
```
Comment (Mu7ammadAbed) [inline client/src/components/Navbar/index.js:64]:
```suggestion
            <Link to="/">
```

**Review by Mu7ammadAbed — CHANGES_REQUESTED** [review]

**Review by Mu7ammadAbed — DISMISSED** [review]

**Review by Mu7ammadAbed — APPROVED** [review]

**Review by muhammadharoun — APPROVED** [review]

**Pull request merged**